Author: Gianfranco Costamagna <locutusofborg@debian.org>
        AsciiWolf
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1575338

Description: this bug seems fixed in pidgin 2.11, this patch can
 (probably) be safely removed on the next update.

--- pidgin-2.10.12.orig/pidgin/data/pidgin.desktop.in.in
+++ pidgin-2.10.12/pidgin/data/pidgin.desktop.in.in
@@ -9,5 +9,4 @@ Terminal=false
 Type=Application
 Categories=Network;InstantMessaging;
 @USES_MM_CHAT_SECTION@
-X-MessagingMenu-UsesChatSection=true
 
